src/components/DatNumber.js

```diff
@@ -16,9 +16,11 @@ export default class DatNumber extends Component {
     path: PropTypes.string,
     label: PropTypes.string,
     labelWidth: PropTypes.number,
+    customLabelWidth: PropTypes.number,
```

Comment:
Is this used at all?

Reply:
Hmm, let me check out if I'm using it at all in my app

Reply:
It's only in the PropTypes, but is never used anywhere else in DatNumber on your branch

Reply:
Ohh okay if it's not used in the actual component then can definitely remove. I'll do it. Thanks!

Reply:
@claus removed